### DISCUSSION TITLE
Is it possible to advertise a webserver using MDNS?

### DISCUSSION THREAD

**Discussion by dburr** [HomeSpan/HomeSpan#226] [Q&A]
I'm working on a HomeSpan project that includes a WebServer for viewing status information (above and beyond what can be seen in the Home app) as well as configuring various settings (again, above and beyond what can be configured in the Home app.) Is there any way I can advertise this over mDNS? I assume that if I include `MDNS.h` and set up mDNS by calling `MDNS.begin()` in my code, that it would conflict with HomeSpan's mDNS advertising (if it even works at all)?

**Comment by HomeSpan** [comment]
No need to provide additional advertising.  You simply need to start a WebServer on a port that is different from the one HomeSpan uses, and the name of your host will be broadcast for use with the WebServer as well as with HomeSpan.

See my [Programmable Hub](https://github.com/HomeSpan/ProgrammableHub) example for details and suggestions on how to fully integrate a web server, change parameters, etc.
